src/content/en/updates/2017/06/budget-api.md

```diff
@@ -0,0 +1,189 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Budget API allows developers to perform background actions without notifying users, enabling use cases like silent push.
+
+{# wf_updated_on: 2017-06-07 #}
+{# wf_published_on: 2017-06-07 #}
+{# wf_tags: javascript,push,budget,api,origintrials #}
```

Comment:
please either add the missing tags to `src/data/commonTags.json` or choose ones that are already there.